### ISSUE TITLE
Steps layering (possibly z-index-related) display bug

### ISSUE THREAD

**Issue by brandonmcconnell** [techniq/svelte-ux#417]
I am using `Steps` and copied the Pagination Integration example from the docs verbatim but see this:

![](https://github.com/techniq/svelte-ux/assets/5913254/463386d4-5802-4027-bff5-898b6ec0ab02)

It appears this resolves if the `::before` parent (where the `[counter-increment:step]` utility is applied) also receives a z-index utility like `z-1`. `isolate` on the `Steps` root would also prevent needing to fight against the z-index layering later.

---

I am running svelte-ux v~0.70.0

**Comment by techniq** [comment]
Hey @brandonmcconnell, that's interesting.  I wonder if there is also some external styling bleeding in, or an older version of tailwind causing a conflict (I had this initially since the component leverages `size-`, but I can't think of anything stacking/z-index related here.

I know the Daisy UI implementation used `z-index`, and I'm fine with adding it here with isolate to keep it scoped, but curious what might be the difference between the docs and your project.

![image](https://github.com/techniq/svelte-ux/assets/177476/20687e45-ebdd-4f8e-9d69-d329db734197)

**Comment by brandonmcconnell** [comment]
@techniq Yeah, I'm curious about that too. I'll investigate some more and respond to this thread later this week if I find something. Thanks.

**Comment by techniq** [comment]
Closing for now as we haven't been able to reproduce
